src/z3c/jbot/tests/test_five.py

```diff
@@ -1,4 +1,4 @@
-import common
+from . import common
 import unittest
```

Comment:
Please swap the order of these imports.  As per PEP-8:

- stdlib imports first
- 3rd-party libraries next
- this package's imports last

(with a blank line between the sections, assuming they're not empty)

Reply:
if this is a thing, shouldn't there be an isort configuration in `setup.cfg` then?

Reply:
Ok, I added a setup.cfg, run isort (I hope using the right configuration) and also did a quick overall manual pep8 check (and added some missing bare exceptions), additional I was using `black -S -L 79`. This should have caught the most. Better now? I would prefer to use isort and black all over the place, this would save a lot of time and discussion :)

Reply:
I don't use isort myself for reasons.

Mixing wholesale style fixes with unrelated changes is usually a bad idea (makes review harder).

Reply:
Well. True. Better to have a good isort cfg first. Sorting imports and formatting code isn't a job a human should bother.